var/spack/repos/builtin/packages/dataspaces/package.py

```diff
@@ -3,75 +3,33 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import six
-
 from spack import *
 
 
-def is_string(x):
-    """validate a string"""
-    try:
-        return isinstance(x, six.string_types)
-    except ValueError:
-        return False
-
-
-class Dataspaces(AutotoolsPackage):
-    """an extreme scale data management framework."""
-
-    homepage = "http://www.dataspaces.org"
-    url      = "https://dataspaces.rdi2.rutgers.edu/downloads/dataspaces-1.6.2.tar.gz"
-
-    version('1.8.0', sha256='7f204bb3c03c2990f5a2d76a29185466b584793c63ada03e5e694627e6060605')
```

Comment:
Hi, @bernhardkaindl. Thanks for taking the time to review this and your comments.

First, on the matter of the tests. Those are not supposed to fail, and I'm concerned that they all did. What was your test environment? To me, it looks like it was probably ultimately a failure of libfabric to initialize the network, but I'm not positive.

Second, I think your comments on legacy support are likely wise. I found the doc you referenced, and I will see what it would take to integrate the two build systems.